src/components/explorer/ApiExplorer.tsx

```diff
@@ -2,13 +2,17 @@ import React from 'react';
 
 import GraphiQL from 'graphiql';
 import 'graphiql/graphiql.css';
+import { Helmet as Head } from 'react-helmet';
 
-export default function () {
+export default () => {
   return (
     <div style={{ height: '100vh' }}>
+      <Head>
+        <title>API Explorer - Cirrus CI</title>
+      </Head>
       <GraphiQL
         schema={undefined}
-        query={`
+        query={`\
```

Comment:
Seems like a typo here?

Reply:
Nope - this just stops the query from appearing to start on line 2 instead of 1